Setting symbol versions as defined in mapfile-vers

The original Makefile ignores the symbol version definitions

--- libXpm-3.5.13/src/Makefile.am.0	2019-12-13 05:51:40.000000000 +0000
+++ libXpm-3.5.13/src/Makefile.am	2022-03-24 09:52:26.208953021 +0000
@@ -5,7 +5,8 @@
 AM_CPPFLAGS = -I$(top_srcdir)/include/X11/
 AM_CFLAGS = $(CWARNFLAGS) $(XPM_CFLAGS)
 
-libXpm_la_LDFLAGS = -version-number 4:11:0 -no-undefined
+libXpm_la_LDFLAGS = -version-number 4:11:0 -no-undefined \
+	-Wl,-M../../../mapfile-vers
 libXpm_la_LIBADD =  $(XPM_LIBS)
 
 libXpm_la_SOURCES =					\
